src/main/resources/admin/tools/dashboard/dashboard.es6

```diff
@@ -46,16 +60,18 @@ exports.get = function(req) {
  * @return {{pageContributions: *, body: *}}
  */
 function renderPart(req) {
-  const datasetMap = getDataset()
-  const dataQueries = getDataQueries(datasetMap)
+  const datasetMap = oldGetDataset()
```

Comment:
Suggestion: Eliminate `oldGetDataSet` altogether, and replace `oldGetDataQueries` with the following --

```javascript
const getContentByType = (contentType) => { ... }
const prepareDataQueries = (dataqueries, contentDatasets) => { ... }

function renderPart () {
  const contentDatasets = getContentByType('dataset')
  const dataQueries = prepareDataQueries(getContentByType('dataquery'), contentDatasets)
  const contentWithDataSources = ... 
  ...
}
```

Reply:
Wont bother spending time fixing old code that is going to be deleted. If it were to stay, I'd do it, but a lot of that old dataquery/dataset code is a rats nest anyways 🤷‍♂️